tests/high_performance_vm_test.go

```diff
@@ -0,0 +1,106 @@
+package tests
+
+import (
+	"strings"
+	"time"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"github.com/google/goexpect"
+	"kubevirt.io/kubevirt/pkg/kubecli"
+	"kubevirt.io/kubevirt/tests"
+)
+
+var _ = Describe("High performance vm test", func() {
+	/*
+	* This test includes the features:
+	* 1. Headless 
+	* 2. Support memory over commitment
+	*/
+
+	const (
+		virtRawVMFilePath          = "tests/manifests/virt-testing-vm.yml"
+		graphicDeviceOffStr        = "Autoattach Graphics Device:  false"
+		vncErr                     = "Can't connec:qt to websocket (400): No graphics devices are present"
+		serialConsoleOutput        = "login as 'cirros' user"
+		overcommitGuestOverheadStr = "Overcommit Guest Overhead:  true"
+		memoryOvercommit           = "true"
+		headless                   = "false"
+		consoleTimeout             = 180*time.Second 
+	)
+	
+	virtClient, err := kubecli.GetKubevirtClient()
+	registryDisk := tests.RegistryDiskFor(tests.RegistryDiskCirros)
+	tests.PanicOnError(err)
+	
+	Context("Headless vm test", func() {
+		headlessDstVMFilePath := "/tmp/headlesstest-vm.json"
+		headlesstestVMName := "headlesstest"
+
+		It("Create headless VM", func() {
+			ProcessTemplateWithParameters(virtRawVMFilePath, headlessDstVMFilePath, "VM_NAME="+headlesstestVMName, "AUTO_GRAPHIC_DEVICE="+headless, "IMAGE_NAME="+registryDisk, "VM_APIVERSION="+vmAPIVersion)
```

Comment:
s/Process.../tests.Process.../

you will have to add `tests` prefix to all names defined in `tests` package.